SwiftUIDZ/ContentView.swift

```diff
@@ -7,18 +7,177 @@
 
 import SwiftUI
 
+// Экран плеера
 struct ContentView: View {
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let stopButtonName = "Stop"
+        static let playButtonName = "Play"
+        static let shareButtonName = "Поделиться"
+        static let shareQuestionText = "Вы уверены?"
+        static let downloadButtonName = "Скачать"
+        static let downloadResultText = "сохранен в папку загрузки"
+        static let chevronForwardName = "chevron.forward.2"
+        static let chevronBackwardName = "chevron.backward.2"
+        static let yesText = "Да"
+        static let noText = "Нет"
+        static let defaultLongWidthNumber: CGFloat = 100
+        static let defaultShortWidthNumber: CGFloat = 50
+        static let defaultHeightNumber: CGFloat = 50
+        static let defaultAspectRatioNumber: CGFloat = 1
+        static let defaultCornerRadiusNumber: CGFloat = 10
+    }
+    
+    // MARK: - Public Properties
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            songImage
+            nameSongText
+            HStack {
+                currentTimeText
+                Spacer()
+                allTimeText
+            }
+            playerSlider
+            HStack {
+                backSongButton
+                playButton
+                stopButton
+                nextSongButton
+            }
+            HStack {
+                downloadSongButton
+                shareSongButton
+            }
         }
+    }
+    
+    // MARK: - Private Properties
+    
+    @ObservedObject private var viewModel = PlayerViewModel()
```

Comment:
```suggestion
    @StateObject private var viewModel = PlayerViewModel()
```
Теперь так пишем см. в канал важное